### PR TITLE
[fix] revised typo

### LINKED ISSUE #3496: Found a typo in the Maintainer's Contributing Guide
I was inspired by @sarah11918's speech in the current Discord community call, so I revised a typo.

### PR DESCRIPTION
**Pull request by CBID2**
#### What kind of changes does this PR include?

- Minor content fixes (broken links, typos, etc.)

#### Description

- Revises a typo in the contributing guide. 
- Closes #3496

### CODE REVIEW

**Review by ElianCodes — APPROVED** [review]
NWTWWHB 👌

@sarah11918, I'll wait for your approval to merge!

**Comment by CBID2** [comment]
> NWTWWHB 👌
> 
> 
> 
> @sarah11918, I'll wait for your approval to merge!

Thanks @delucis! :) What does "NWTWWHB" mean?

File: contributor-guides/astro-maintainers-guide-to-docs.md
Comment (delucis) [inline contributor-guides/astro-maintainers-guide-to-docs.md:54]: Whoops looks like this is missing a closing `_` on the italic formatting here:
```suggestion
- Leave the message **LGTM** (_Looks good to me_, _Let's get that money_, _Let's get to merging_...) on a PR you sign off on... but don't feel comfortable merging yourself. For example, a proposed new section or explanation looks good to you, but you suspect that other maintainers will also want the opportunity to review it.
```

@CBID2 “NWTWWHB” is a silly docs initialism we use sometimes meaning “Not Worse Than What We Had Before”. We use it to remind ourselves that things don’t need to be perfect, they just need to improve on where we are currently.

Comment (CBID2) [reply contributor-guides/astro-maintainers-guide-to-docs.md:54]: Ohhh that make sense

File: contributor-guides/astro-maintainers-guide-to-docs.md
Comment (sarah11918) [inline contributor-guides/astro-maintainers-guide-to-docs.md:54]:
```suggestion
- Leave the message **LGTM** (_Looks good to me_, _Let's get that money_, _Let's get to merging_...) on a PR you sign off on... but don't feel comfortable merging yourself. For example, a proposed new section or explanation looks good to you, but you suspect that other maintainers will also want the opportunity to review it. If your approval concerns only a specific aspect of the PR (e.g. you have verified a code sample is correct; you agree with the intention, if not the exact wording etc.) then please be specific about *what* looks good to you. This is to help the maintainer who will assume responsibility for merging the PR know which aspects have been reviewed, and also to not give the PR author "false hope" that the entire PR is ready to merge!
```

**Review by sarah11918 — APPROVED** [review]
The perfect excuse for me to slip in something I've had to say to people a few times recently!

**Comment by CBID2** [comment]
> The perfect excuse for me to slip in something I've had to say to people a few times recently!

Lol 😆

**Review by Yan-Thomas — APPROVED** [review]

**Pull request merged**
